src/main/webapp/WEB-INF/tags/instructor/results/copyPasteModal.tag

```diff
@@ -7,11 +7,13 @@
 <%@ attribute name="courseId" required="true" %>
 <%@ attribute name="feedbackSession" required="true" %>
 <%@ attribute name="selectedSection" required="true" %>
+<%@ attribute name="selectedSectionDetail" required="true" %>
 <form id="csvToHtmlForm">
   <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${courseId}">
   <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_SESSION_NAME%>" value="${feedbackSession}">
   <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" value="${data.account.googleId}">
   <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYSECTION%>" value="${fn:escapeXml(selectedSection)}">
+  <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYSECTIONDETAIL%>" value="${fn:escapeXml(selectedSectionDetail)}">
```

Comment:
Why do we need need this here?

Reply:
This is necessary for the `Non-English characters not displayed properly in the downloaded file? Click here.` part in the sessions result page to show the copy paste modal character in the same way as downloaded csv file when filter by a specific section.